[Bug] Expected non-nullable value [NullSafeMutableLiveData]

## 현상

```
* What went wrong:
Execution failed for task ':feature:detail:lintDebug'.
> Lint found errors in the project; aborting build.
  /home/runner/work/Moop-Android/Moop-Android/feature/detail/src/main/java/soup/movie/detail/DetailViewModel.kt:60: Error: Expected non-nullable value [NullSafeMutableLiveData]
              _favoriteUiModel.postValue(repository.isFavoriteMovie(movie.id))
                                         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
- Lifecycle 라이브러리의 Lint Rule 오류로 잘못 검출되는 현상으로 보입니다.

## Link

- https://issuetracker.google.com/issues/169249668

--- a/feature/detail/src/main/java/soup/movie/detail/DetailViewModel.kt
+++ b/feature/detail/src/main/java/soup/movie/detail/DetailViewModel.kt
@@ -57,7 +57,8 @@ class DetailViewModel @Inject constructor(
         this.movie = movie
         _headerUiModel.value = HeaderUiModel(movie)
         viewModelScope.launch {
-            _favoriteUiModel.postValue(repository.isFavoriteMovie(movie.id))
+            //FIXME: Elvis operator (?:) is used because of lint rule error https://issuetracker.google.com/issues/169249668
+            _favoriteUiModel.postValue(repository.isFavoriteMovie(movie.id) ?: false)
             val minDelay = async { delay(500) }
             val loadDetail = async {
                 loadDetail(movie)

